pmoves/tools/_secrets_common.py

```
@@ -0,0 +1,140 @@
+    if lowered.startswith("placeholder_"):
+        return True
+    # Example domain checks (URLs and emails)
+    if lowered.endswith("@example.com") or lowered.endswith(".example.com"):
+        return True
+    try:
+        candidate = normalized if "://" in normalized else f"https://{normalized}"
+        host = (urllib.parse.urlparse(candidate).hostname or "").lower()
+        if host in {"example.com", "www.example.com"}:
+            return True
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)